docs/deploying-to-azure-containerapps.md

```diff
@@ -730,6 +731,72 @@ az staticwebapp secrets list  \
   --output table
 ```
 
+# Load Testing
+
+Now time to add some load to the application.
+This way, we will be able to see the auto-scaling in Azure Container Apps.
+
+To add some load to an application, you can do it locally using [JMeter](https://jmeter.apache.org), but you can also do it remotely on Azure using [Azure Load Testing](https://azure.microsoft.com/services/load-testing) and JMeter.
+Azure Load Testing is a fully managed load-testing service built for Azure that makes it easy to generate high-scale load and identify app performance bottlenecks.
+It is available on the [Azure Marketplace](https://azuremarketplace.microsoft.com).
+For that, we need to go the 
+
+To use Azure Load Testing, go to the [Azure Portal](https://portal.azure.com), search for the Marketplace and look for "_Azure Load Testing_"  in the Marketplace.
+Click on "_Create_":
+
+![load-testing-1-marketplace](../images/load-testing-1-marketplace.png)
+
+Create a load testing resource by giving it a name (`super-heroes-load-testing`), a location, and a resource group.
```

Comment:
The load testing resource name needs to be unique. When I tried to create one it told me the name was already taken.